front/components/vaults/VaultSideBarMenu.tsx

```diff
@@ -205,13 +298,15 @@ const DATA_SOURCE_OR_VIEW_SUB_ITEMS: {
 };
 
 const VaultDataSourceOrViewItem = ({
+  item,
   owner,
   vault,
-  item,
+  viewType,
 }: {
+  item: DataSourceOrViewInfo;
   owner: LightWorkspaceType;
   vault: VaultType;
-  item: DataSourceOrViewInfo;
+  viewType: "data_sources" | "data_source_views";
```

Comment:
Should we move the `DataSourceOrView` type from swr to types and reuse it here?

Reply:
Indeed this one could live in `types`. Let me move it.